renderdoccmd/renderdoccmd_android.cpp

```diff
@@ -28,6 +28,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <string>
+#include "../os/android/android_specific.h"
```

Comment:
I will rewrite this without needing to access the core library, using the better method.